src/lib/y2storage/autoinst_profile/drive_section.rb

```diff
@@ -162,6 +167,15 @@ def init_from_disk(disk)
         true
       end
 
+      # Device name to be used for the real MD device
+      #
+      # @see PartitionSection#name_for_md for details
+      #
+      # @return [String] MD RAID device name
+      def name_for_md
+        partitions.first.name_for_md
```

Comment:
No if the profile is correct. We have a separate PBI to add proper error reporting for cases in which we receive non-sense profiles. But I will add a comment to make sure we don't skip this particular point when adding such error reporting.